### ISSUE TITLE
Unload Combine - Autodrive driver doesn't stop under combine pipe to unload and loops repeatedly. 

### ISSUE THREAD

**Issue by tracrr** [Stephan-S/FS22_AutoDrive#656]
Autodrive combine autoload refuses to work and actually unload the combine. I've also captured the settings of autodrive too. https://youtu.be/YWiK2JkOciU - FYI mute the video before watching, recorded while watching stream. 

Things I've tried to fix this:

Complete uninstall and re-install of both autodrive and courseplay.

Deleted my old save and started all over again.

Tried modded and unmodded trailers and tractors all in different combinations.

Attempted a separate combine unload on a different map with the same issue.

A lot of googling to try and resolve this also.

All I've been able to find is bugs from 6 months ago which has been fixed with new releases.
Thanks for your hard work :)

**Comment by GTA3Fan** [comment]
Have a similar issue, only mine is with a Semi and Trailer. The truck lines up with the chute as it turns but sense it is semi it turns sharper and the trailer doesn't follow properly and just gets stuck in this loop until I intervene and correct it. 

Just saw this post as I was about to generate one myself, I'll get a video again when it happens.

**Comment by Farmerboy1124** [comment]
I have this issue too! The tipper just runs right under the pipe like it is supposed to but it never stops. It is like the combine goes missing for a minute and then pops back up after the tractor has driven by completely, causing the tractor to have to loop around and recalculate its path but then do the same thing again and never actually unloads
Here is the Autodrive mod file I am using.
[FS22_AutoDrive.zip](https://github.com/Stephan-S/FS22_AutoDrive/files/9677256/FS22_AutoDrive.zip)
 the combine.

**Comment by rohne83** [comment]
I'd suppose enabling debug for pathfinding (in AD settings) and then taking a screenshot of the calculated path to the combine. Think that might help to understand why the unloader is going too far in the first place.

@GTA3Fan's issue looks a bit different to me though. This has rather to do with the turn radius of the trailer (the first bin is filled but the trailer reaches the combine in such an angle that the second bin is not hit by the pipe). Placing the truck manually behind the combine so it only has to go about straight forward to the combine and reenable AD there might result in a completely different picture.
But I'm wondering whether this kind of trucks are used to actively unload combines IRL at all. Every scene I saw from US and Australian harvests with such trailers, the trucks where parked at the field edge and filled by an auger wagon.

**Comment by Farmerboy1124** [comment]
Actually I believe I figured it out. I had just added the manual unload mod in and it was interfering with all the worker unloading. So I guess it’s good to know that mod messes with worker unloading🤷🏻

Sent from my iPhone

On Oct 14, 2022, at 7:23 AM, rohne83 ***@***.***> wrote:

﻿

I'd suppose enabling debug for pathfinding (in AD settings) and then taking a screenshot of the calculated path to the combine. Think that might help to understand why the unloader is going too far in the first place.

@GTA3Fan<https://nam12.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FGTA3Fan&data=05%7C01%7C%7C5985d999855947a5321008daaddef36c%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C638013470334185501%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=nPmyQf9EhiBNrCdXhKdvKVu%2BcV%2FeeeuOIuaKSaECe3I%3D&reserved=0>'s issue looks a bit different to me though. This has rather to do with the turn radius of the trailer (the first bin is filled but the trailer reaches the combine in such an angle that the second bin is not hit by the pipe). Placing the truck manually behind the combine so it only has to go about straight forward to the combine and reenable AD there might result in a completely different picture.
But I'm wondering whether this kind of trucks are used to actively unload combines IRL at all. Every scene I saw from US and Australian harvests with such trailers, the trucks where parked at the field edge and filled by an auger wagon.
